### PR TITLE
Update dependencies & Fix missing imports

### PR DESCRIPTION
**Pull request by clemtrek**

### CODE REVIEW

File: bench/run.js [hunk 1]
```diff
@@ -2,6 +2,7 @@
 const childProcess = require('child_process');
 const path = require('path');
 const fs = require('fs');
+const Promise = require('bluebird');
```
Comment (sindresorhus) [inline bench/run.js]: Don't do unrelated changes.

**Comment by sindresorhus** [comment]
The PR needs a more descriptive title.

File: bench/run.js (hunk 1 above)
Comment (clemtrek) [reply bench/run.js]: Shall I remove them and add them in another PR?

**Comment by novemberborn** [comment]
`@ava/init` will land in https://github.com/avajs/ava/pull/1738. I've removed the deprecation message from npm since it's not actually useful for anybody installing AVA.

Before you make a new PR for the promise change, could you elaborate on why it's necessary?
